exercises/sieve/test/test_sieve.c

```diff
@@ -0,0 +1,114 @@
+#include <string.h>
+#include "vendor/unity.h"
+#include "../src/sieve.h"
+
+primesArray_t result_array;
+
+// Individual Tests
```

Comment:
I like to separate the "Tests" from the helper functions...  since there are no helper functions in this case I can delete the comment